### ISSUE TITLE
Selector for direct child elements

### ISSUE THREAD

**Issue by kowach** [Imangazaliev/DiDOM#35]
I wanted to find all direct children under `$element->find( '>*' )` but it returned all children and children under children in recursion. 

    $document->loadHtml("<div><span></span></div>")->find( '>*' );
    // returns two elements div and span, while jQuery('>*') returns only div

`>*` is compiled to `//*/*` but it should be compiled to `/*/*` which gives direct child elements.

**Comment by Imangazaliev** [comment]
You can go the other way:
```php
use DiDom\Query;

...

$document->loadHtml("<div><span></span></div>");
$elements = $document->xpath(Query::cssToXpath('* > *', '/'));
```

**Comment by Imangazaliev** [comment]
Implemented in version [1.8](https://github.com/Imangazaliev/DiDOM/releases/tag/1.8).
